### PR TITLE
Add timestream clients

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: docs/clients/timestream-query.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: client
+category: clients
+name: Timestream Query
+package: async-aws/timestream-query
+---
+
+## Usage
+
+### Send a message
```
Comment (jderusse) [inline docs/clients/timestream-query.md:10]: execute a query

File: src/Core/src/AwsClientFactory.php [hunk 2]
```diff
@@ -439,6 +441,32 @@ public function stepFunctions(): StepFunctionsClient
         return $this->serviceCache[__METHOD__];
     }
 
+    public function timestreamQuery(): TimestreamQueryClient
```
Comment (jderusse) [inline src/Core/src/AwsClientFactory.php:444]: Add entries in changelog

File: src/Service/TimestreamQuery/src/TimestreamQueryClient.php [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        return new JsonRpcAwsErrorFactory();
+    }
+
+    protected function getEndpointMetadata(?string $region): array
+    {
+        if (null === $region) {
+            $region = Configuration::DEFAULT_REGION;
+        }
+
+        switch ($region) {
```
Comment (jderusse) [inline src/Service/TimestreamQuery/src/TimestreamQueryClient.php:146]: 🤔 generator could be improved to remove this empty switch

**Review by jderusse — COMMENTED** [review]
thank you for this new service 🎉

**Comment by GrahamCampbell** [comment]
Done. 🎉

**Review by jderusse — APPROVED** [review]

**Pull request merged**